core/templates/pages/exploration-editor-page/preview-tab/templates/preview-set-parameters-modal.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview Unit tests for the PreviewSetParametersModalComponent.
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { PreviewSetParametersModalComponent } from './preview-set-parameters-modal.component';
+
+class MockActiveModal {
+  close(): void {
+    return;
+  }
+
+  dismiss(): void {
+    return;
+  }
+}
+
+describe('Preview Set Parameters Modal Component', function() {
+  let component: PreviewSetParametersModalComponent;
+  let fixture: ComponentFixture<PreviewSetParametersModalComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PreviewSetParametersModalComponent,
+      ],
+      providers: [{
+        provide: NgbActiveModal,
+        useClass: MockActiveModal
+      }],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PreviewSetParametersModalComponent);
+    component = fixture.componentInstance;
+
+    TestBed.inject(NgbActiveModal);
+  });
+
+  it('should create', () => {
```

Comment:
Please add a comment as to why there are no other tests here

Reply:
Done.